src/utils/validators.ts

```diff
@@ -55,11 +55,18 @@ export const maxParticipantsValidator = (
 };
 
 export const nameValidator = (name: string): string[] | undefined => {
-  const nameReg = /^[a-zæøåA-ZÆØÅ ]+$/; //! note the last space
+  // \p{L} -> letter in any language
+  // \p{Zz} -> allows space
+  // 1,30 is the valid range
+  // must be an range or any name over one char gets an error
+  const nameReg = /^[\p{L}\p{Zs}]{1,30}$/u;
```

Comment:
Should also allow `-`, at least in the last name (ex: Marius Møller-Hansen should be valid).

Reply:
Well, I left any of these characters out as it will allow for name such as `---`

Reply:
fixed now